### PR TITLE
feat(codegen): added default imports for hashKey helper, design-system and input types

### PR DESCRIPTION
**Pull request by gpuente**

### CODE REVIEW

File: packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model-module/customTest.esm.t [hunk 1]
```diff
@@ -24,7 +30,13 @@ describe('<%= h.changeCase.pascal(module) %> Operations', () => {
 
 <% actions.forEach(action => { _%>
     it('should handle <%= h.changeCase.camel(action.name) %> operation', () => {
-        const input = generateMock(z.<%= h.changeCase.pascal(action.name) %>InputSchema());
+        // generate a random id
+        // const id = documentModelUtils.hashKey();
```
Comment (froid1911) [inline packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model-module/customTest.esm.t:34]: can be removed?

File: packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-editor/editor.esm.t [hunk 2]
```diff
@@ -6,9 +6,19 @@ unless_exists: true
...
+import { utils as documentModelUtils } from 'document-model/document';
+import { Button } from '@powerhousedao/design-system';
+import { utils as documentModelUtils } from 'document-model/document';
 
 export type IProps = <% if(!documentTypes.length){ %>EditorProps<unknown, Action><% } else { %><% documentTypes.forEach((type, index) => { _%>EditorProps<<%= documentTypesMap[type] %>State, <%= documentTypesMap[type] %>Action, <%= documentTypesMap[type] %>LocalState%>%>><% if(index < documentTypes.length - 1){ %> | <% }%><% }); _%> <% } %>;
 
 export default function Editor(props: IProps) {
-    return <></>;
+    // generate a random id
+    // const id = documentModelUtils.hashKey();
```
Comment (froid1911) [inline packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-editor/editor.esm.t:16]: can be removed too?

**Review by froid1911 — APPROVED** [review]

File: packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-editor/editor.esm.t [hunk 3]
```diff
@@ -6,9 +6,19 @@ unless_exists: true
 <% documentTypes.forEach(type => { _%>
 import { <%= documentTypesMap[type] %>State, <%= documentTypesMap[type] %>Action, <%= documentTypesMap[type] %>LocalState, actions } from "../.<%= documentModelsDir %>/<%= h.changeCase.param(documentTypesMap[type]) %>";
 %><% }); _%>
+import { utils as documentModelUtils } from 'document-model/document';
+import { Button } from '@powerhousedao/design-system';
+import { utils as documentModelUtils } from 'document-model/document';
```
Comment (acaldas) [inline packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-editor/editor.esm.t]: Duplicated

Comment (gpuente) [reply packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-editor/editor.esm.t]: fixed: [afdc0b5](https://github.com/powerhouse-inc/powerhouse/pull/467/commits/afdc0b529f03a6e828ebebc9055847042409c16d)

File: packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model-module/customTest.esm.t (hunk 1 above)
Comment (gpuente) [reply packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model-module/customTest.esm.t:34]: I can remove it, but I putted this one here so when we generate the test file and the editor, you have an example of how to generate a random ID.

File: packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-editor/editor.esm.t (hunk 2 above)
Comment (gpuente) [reply packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-editor/editor.esm.t:16]: same here

**Pull request merged**